core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -696,4 +696,12 @@ package object matryoshka extends CofreeInstances with FreeInstances {
 
   implicit def toFreeOps[F[_], A](a: Free[F, A]): FreeOps[F, A] =
     new FreeOps[F, A](a)
+
+  implicit def equalTEqual[T[_[_]], F[_]: Functor](implicit T: EqualT[T], F: Delay[Equal, F]):
+      Equal[T[F]] =
+    T.equalT[F](F)
```

Comment:
Could've used this yesterday! Now that I know it exists, let me just go write some more quasar code that will need to be cleaned up when this is merged. ;)